docs/openstack/octavia-adoption.md

```diff
@@ -0,0 +1,122 @@
+# Octavia adoption
+
+Migrating from a Director based to an OpenShift Octavia deployment
+presents some special challenges. These require that some of the
+original components continue to run until all processes that reference
+them are no longer running or have been altered to reference the new
+Octavia services running in OpenShift.
+
+
+## Prerequisites
+
+Previous Adoption steps have been completed, including:
+ * MariaDB
+ * Keystone
+ * OVN
+ * Neutron
+ * Nova
+ * Glance
+ * Barbican (optionally)
+
+> The Octavia database should have been included in the database transfer for the MariaDB adoption process. The OVN db should also have been migrated during the OVN migration process.
+
+## Start the Redis Pod (optional)
+
+If Octavia is configured to use Taskflow for task coordination, then Redis should be enabled prior to migration. To start Redis, patch the OpenStackControlPlane:
+
+```
+oc patch openstackcontrolplane openstack --type=merge --patch '
+spec:
+  redis:
+    enabled: true
+	templates:
+    - something-octavia-specific-tbd
+'
+```
+
+## Shutdown Director Deployed Octavia Services
```

Comment:
I forgot you added this here already and added it in #174. A few questions:

1. Shouldn't we stop Octavia services before MariaDB data copy already in order prevent inconsistencies?
2. Your list of services below does not contain the driver agent server and the rsyslog services. I assume you did that on purpose, but I wanted to highlight that in case it was missed somehow.